faq.html

```diff
@@ -263,6 +264,14 @@ <h2 id="instructors">Instructor Training</h2>
     </p>
   </dd>
 
+  <dt id="why-do-we-refer-to-our-students-as-learners">Why do we refer to students as 'learners'?</dt>
+  <dd>
+    <p>
+     We settled on "learner" because most professors (who may be learners)
```

Comment:
On Wed, Mar 25, 2015 at 06:55:11AM -0700, Greg Wilson wrote:
> Would be more accurate to say: We use the terms 'instructor' and
> 'learner' because words like 'professor' and 'student' are job
> descriptions as well as descriptions of roles, and we don't want to
> give the impression that you can only attend a workshop if you're
> what a university would call a student.

Ah, this is a much better reason :).